src/css/themes/po-theme-default.css

```
@@ -1551,28 +1551,53 @@ po-textarea {
   --color-upload-drag-drop-area-color-disabled: var(--color-neutral-light-60);
 
   --color-upload-file-restrictions-color-text: var(--color-neutral);
-
-  /*------------------------------------*\
+}
+/*------------------------------------*\
     WIDGET
   \*------------------------------------*/
 
+po-widget {
+  --padding: 1em;
+  --border-radius: var(--border-radius-md);
+  --border-color: var(--color-neutral-light-20);
```

necessário manter customização atual:
```suggestion
  --border-color: var(--color-neutral-light-80);
```